crates/data_components/src/cdc.rs

```diff
@@ -31,6 +31,9 @@ pub enum CommitError {
     UnableToCommitChange {
         source: Box<dyn std::error::Error + Send + Sync>,
     },
+
+    #[snafu(display("Change committer not exist"))]
+    CommitterMissing {},
```

Comment:
The message and the error object name should be consistent.

Reply:
Error message updated